internal/model/sql/postgres/repository.go

```diff
@@ -635,6 +641,7 @@ func GetIncidentStatusFilterQuery() string {
 		, CASE WHEN commander_email IS NULL THEN '' ELSE commander_email END commander_email
 	FROM incident
 	WHERE status IN ($1, $2)
+	AND channel_type <> ($3)
```

Comment:
Why this condition in this function?

Reply:
This function  returns all the incidents that we want to notify in cron-notify. In that case, I don't want the silent incident to be reported on the channel.

Reply:
ok!